MIGRATION.MD

```diff
@@ -0,0 +1,234 @@
+# Migration from 1.x to 2.0
+
+The main changes introduced in SDK 2.0 compared to 1.x are:
+
+1. All the products (RUM, Trace, Logs, etc.) are now extracted into different modules. That allows you to integrate only what is needed into your application.
+
+In comparison with version 1.x where all products were in a single artifact `com.datadoghq:dd-sdk-android:x.x.x`, now you need to adopt the following artifacts instead:
```

Comment:
We might want to include OkHttp artifact too here as it's one that almost everyone will use

Reply:
Indeed, I completely forgot that in v1 OkHttp instrumentation doesn't live in the dedicated module.